Documentation/CRDs/Shared-Filesystem/ceph-filesystem-crd.md

```diff
@@ -135,13 +135,13 @@ spec:
     # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
     peers:
       secretNames:
-        - secondary-cluster-peer
-    # specify the schedule(s) on which snapshots should be taken
-    # see the official syntax here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-schedules
+      # Add the secret name if it already exists else specify the empty list here.
+      # specify the schedule(s) on which snapshots should be taken
```

Comment:
Don't change this format, 
It's for the `snapshotSchedules`

Reply:
@parth-gr sure